docker/docker-compose.1804.53.yaml

```diff
@@ -0,0 +1,28 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: swift-nio-ssl:18.04-5.3
+    build:
+      args:
+        ubuntu_version: "bionic"
```

Comment:
actually, this also can't be right because there's no 5.3 docker images yet because it's not released. This needs to use `base_image=...` instead of `ubuntu_version` & `swift_version`